<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titre">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paiement</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_carte">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>170</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de carte</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_fin">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>220</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>date de fin validit√© (MM/AA)</string>
    </property>
   </widget>
   <widget class="QLabel" name="Crypto">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>270</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cryptogramme visuel </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_numCarte">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Crypto">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_annuler">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_valider">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Retour">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour moyens de paiement</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_mois">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_annee">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="amount">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Montant a payer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_amount">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>=============================================================================</string>
    </property>
   </widget>
   <widget class="QLabel" name="nomPrenom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom et Prenom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomPrenom">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
